src/buttons/hangmanLetterButton.test.ts

```diff
@@ -40,7 +46,6 @@ describe('hangmanMoreButton', () => {
 			{ embeds: [{ data: { fields: [unknown, { value: string }] } }] }
 		];
 		const afterInfo = lastCall[0].embeds[0].data.fields[1].value;
-		expect(afterInfo).toBeString();
 		expect(afterInfo).not.toEqual(beforeInfo);
```

Comment:
```suggestion
		expect(afterInfo).toBeTypeOf('string');
		expect(afterInfo).not.toEqual(beforeInfo);
```

Reply:
Another instance of using type-checking. `afterInfo` is already guaranteed to be a string.